src/main/java/org/jabref/preferences/ai/EmbeddingModel.java

```diff
@@ -0,0 +1,345 @@
+package org.jabref.preferences.ai;
+
+import org.apache.commons.io.FileUtils;
+
+public enum EmbeddingModel {
```

Comment:
Please add a comment which information was the base to fill this

(Remember someone looking at this part of the code in two years and wonders how to a) check entries for validity and b) how to add new entries)